core/utils.h

```diff
@@ -1946,6 +1946,14 @@ notify(syslog_event_type_t priority, bool internal, bool synch,
 #define SYSLOG_INTERNAL_CRITICAL_ONCE(...) \
     DODEBUG_ONCE(SYSLOG_INTERNAL_CRITICAL(__VA_ARGS__))
 
+#define MSG_COULDNT_SYNCH_WITH_ALL_THREADS_CODE 42
+
+#define fatal_error(id, sub, ...) \
```

Comment:
How about msg_id and arg_count for more descriptive names

Reply:
I was copying the existing 'SYSLOG' code, but I agree that it took me a while to figure out what those parameters meant...